modules/mod_admin/templates/_admin_menu.tpl

```diff
@@ -80,7 +80,7 @@
 
             {% block search %}
                 <form class="navbar-right navbar-form form-inline" action="{% block search_target %}{% url admin_overview_rsc %}{% endblock %}" method="get">
-                    <input type="hidden" name="qsort" value="{{ q.qsort|escape }}" />
+                    {# <input type="hidden" name="qsort" value="{{ q.qsort|escape }}" /> #}
```

Comment:
Just remove this?

Reply:
Ja, kan er nu wel uit (als dit gedrag goed genoeg is).